src/shared/components/Popover/PopoverDialog.styles.tsx

```diff
@@ -0,0 +1,51 @@
+import styled from '@emotion/styled'
+
+import { colors, sizes } from '@/shared/theme'
+
+import { Popover } from './Popover'
+
+import { Text } from '../Text'
+
+export const MAX_CONTENT_HEIGHT = 219
+
+type Scrollable = {
+  isScrollable: boolean
+}
+
+export const HeaderContainer = styled(Text)<Scrollable>`
+  ${({ isScrollable }) =>
+    isScrollable &&
+    `
+    padding-bottom: ${sizes(4)};
+    border-bottom: 1px solid ${colors.gray[600]};
+  `}
```

Comment:
If we need to conditionally include a block of CSS like this, let's extract the relevant styles to a separate constant. That way we get proper CSS intellisense&linting, I don't think the tooling will be able to understand this string. 

Example:
```tsx
const headerContainerScrollableStyles = css`
  padding-bottom: ${sizes(4)};
  border-bottom: 1px solid ${colors.gray[600]};
`

export const HeaderContainer = styled(Text)<Scrollable>`
  ${({ isScrollable }) => isScrollable && headerContainerScrollableStyles};
`